core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java

```diff
@@ -70,6 +70,11 @@
     private Duration statusRefreshMaxWait = new Duration(1, TimeUnit.SECONDS);
     private Duration infoUpdateInterval = new Duration(3, TimeUnit.SECONDS);
 
+    private boolean interruptStuckSplitTasksEnabled = true;
```

Comment:
nit: I wonder if it will be more readable / convenient if the property naming was aligned, e.g.:

- `interruptStuckSplitTasksEnabled` 
- `interruptStuckSplitTasksWarningThreshold`
- `interruptStuckSplitTasksTimeout`
- `interruptStuckSplitTasksDetectionInterval`

WDYT?